light-client/src/light_client.rs

```diff
@@ -26,23 +26,16 @@ pub struct Options {
     /// and trusted validator set is sufficient for a commit to be
     /// accepted going forward.
     pub trust_threshold: TrustThreshold,
+
     /// How long a validator set is trusted for (must be shorter than the chain's
     /// unbonding period)
     pub trusting_period: Duration,
+
     /// Correction parameter dealing with only approximately synchronized clocks.
     /// The local clock should always be ahead of timestamps from the blockchain; this
     /// is the maximum amount that the local clock may drift behind a timestamp from the
     /// blockchain.
     pub clock_drift: Duration,
-    /// The current time
-    pub now: Time,
```

Comment:
I'm actually using these "Options" in #430 too. I was wondering how this made sense as an option. Much clearer when it isn't 👍